Standards/scs-0001-v1-sovereign-cloud-standards.md

```diff
@@ -118,10 +118,20 @@ embedded in the markdown header.
 
 ### Sections
 
-Each document SHOULD have the following sections:
+#### Standard
+
+Each Standard document MUST have the following sections:
 
 - An _Introduction_ providing context on the document and linking to other relevant materials.
 - A _Motivation_ section which details why this document or the thing it describes is necessary.
+- A section containing the actual standardization decision. The naming for this is up to the standards
+  author, since the flow of the document possibly changes the naming. We RECOMMEND naming the section _Standard_.
```

Comment:
I still have to think about this some more. I'm still not convinced, but I can imagine that you are on to something here.

Reply:
```suggestion
- A section containing the actual standardization decision.
```

Reply:
Changed in 6fb17a19fc38f70a0dda6c5cc9689ff28d0b31e4